src/omniperf_soc/soc_base.py

```diff
@@ -197,6 +203,9 @@ def populate_mspec(self):
             # We need to distinguish MI308X by peeking reported num CUs
             elif self._mspec.cu_per_gpu == "80" or "MI308X" in self.check_arch_override():
                 self._mspec.gpu_model = "MI308X"
```

Comment:
I don't think we need this bit anymore now that we're setting `gpu_model` based on `MI300_CHIP_IDS` map.